Resources/Prototypes/_Moffstation/Entities/Structures/Walls/fence_wrought_iron.yml

```
@@ -0,0 +1,161 @@
+            max: 1
+      - !type:DoActsBehavior
+        acts: [ "Destruction" ]
+  - type: Climbable
+    startClimbSound:
+      collection: FootstepHull
+    finishClimbSound:
+      collection: FootstepHull
+
+- type: entity
+  parent: BaseFenceWroughtIron
+  id: FenceIronStraight
+  suffix: Straight
+  components:
+  - type: Icon
+    sprite: _Moffstation/Structures/Walls/wrought_iron_fence.rsi
+    state: icon_straight
+  - type: Sprite
+    layers:
+    - state: straight
+  - type: Fixtures
+    fixtures:
+      fix1:
+        shape:
+          !type:PhysShapeAabb
+          bounds: "-0.10,-0.5,0.10,0.5"
+        density: 1000
+        mask:
+        - FullTileMask
+        layer:
+        - MidImpassable
+        - LowImpassable
+
+- type: entity
+  parent: BaseFenceWroughtIron
+  id: FenceIronCorner
+  suffix: Corner
+  components:
+  - type: Sprite
+    layers:
+    - state: corner
+  - type: Fixtures
+    fixtures:
+      # needs two shapes to properly handle a triangle corner without weirdness
+      fix1:
+        shape:
+          !type:PhysShapeAabb
+          bounds: "-0.1,-0.1,0.1,0.5"
+        density: 1000
+        mask:
+        - TableMask
+        layer:
+        - TableLayer
+      fix2:
+        shape:
+          !type:PhysShapeAabb
+          bounds: "-0.5,-0.1,-0.1,0.1"
+        density: 1000
+        mask:
+        - TableMask
+        layer:
+        - MidImpassable
+        - LowImpassable
+
+- type: entity
+  parent: BaseFenceWroughtIron
+  id: FenceIronEnd
+  suffix: End
+  components:
+  - type: Icon
+    sprite: _Moffstation/Structures/Walls/wrought_iron_fence.rsi
+    state: icon_end
+  - type: Sprite
+    layers:
+    - state: end
+  - type: Fixtures
+    fixtures:
+      fix1:
+        shape:
+          !type:PhysShapeAabb
+          bounds: "-0.1,-0.5,0.1,0.0"
+        density: 1000
+        mask:
+        - TableMask
+        layer:
+        - MidImpassable
+        - LowImpassable
+
+- type: entity
+  parent: BaseFenceWroughtIron
+  id: FenceIronGate
+  name: wrought-iron fence gate
+  description: The less-pokey way to get past the fence
+  components:
+  - type: Sprite
+    layers:
+    - state: end
+      map: ["enum.DoorVisualLayers.Base"]
+  - type: Fixtures
+    fixtures:
+      fix1:
+        shape:
+          !type:PhysShapeAabb
+          bounds: "-0.5,-0.1,0.5,0.1"
+        density: 1000
+        mask:
+        - TableMask
+        layer:
+        - MidImpassable
+        - LowImpassable
+  - type: InteractionOutline
+  - type: Door
+    openSpriteState: door_opened
+    closedSpriteState: door_closed
+    canPry: false
+    occludes: false
+    changeAirtight: false
+    bumpOpen: false
+    clickOpen: true
+    canCrush: false
+    closeTimeOne: 0
+    closeTimeTwo: 0
+    openTimeOne: 0
+    openTimeTwo: 0
+    openingAnimationTime: 0
+    closingAnimationTime: 0
+    openSound:
+      path: /Audio/Effects/woodenclosetopen.ogg
+    closeSound:
+      path: /Audio/Effects/woodenclosetclose.ogg
```

```suggestion
    openSound:
      path: /Audio/Effects/woodenclosetopen.ogg
      params:
        pitch: 1.0 # or something else here
    closeSound:
      path: /Audio/Effects/woodenclosetclose.ogg
      params:
        pitch: 1.0 # or something else here
```
for pitching the audio

---

I made two copies of the prototype, one with 0.99 for both values and one with 0.01, and they sound identical.
I've tried cranking it up to 10, also, and that seemed identical too.
Maybe it's bugged or something?

IMO, it still sounds pretty reasonable as-is.